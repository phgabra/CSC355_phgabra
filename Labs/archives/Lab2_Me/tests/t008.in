int a = 123;
float b = 4.56;
char c = '7';
int x = 5 + 10;
x = x * 3 / 2 - 1;
char ch = 'a';
string str = "Hello, World!";
