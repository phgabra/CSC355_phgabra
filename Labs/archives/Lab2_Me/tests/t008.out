<T_KEYWORD, 'int'>
<T_IDENTIFIER, 'a'>
<T_OPERATOR, '='>
<T_NUMBER, '123'>
<T_DELIMITER, ';'>
<T_KEYWORD, 'float'>
<T_IDENTIFIER, 'b'>
<T_OPERATOR, '='>
<T_NUMBER, '4.56'>
<T_DELIMITER, ';'>
<T_KEYWORD, 'char'>
<T_IDENTIFIER, 'c'>
<T_OPERATOR, '='>
<T_CHAR_LIT, ''7''>
<T_DELIMITER, ';'>
<T_KEYWORD, 'int'>
<T_IDENTIFIER, 'x'>
<T_OPERATOR, '='>
<T_NUMBER, '5'>
<T_OPERATOR, '+'>
<T_NUMBER, '10'>
<T_DELIMITER, ';'>
<T_IDENTIFIER, 'x'>
<T_OPERATOR, '='>
<T_IDENTIFIER, 'x'>
<T_OPERATOR, '*'>
<T_NUMBER, '3'>
<T_OPERATOR, '/'>
<T_NUMBER, '2'>
<T_OPERATOR, '-'>
<T_NUMBER, '1'>
<T_DELIMITER, ';'>
<T_KEYWORD, 'char'>
<T_IDENTIFIER, 'ch'>
<T_OPERATOR, '='>
<T_CHAR_LIT, ''a''>
<T_DELIMITER, ';'>
<T_KEYWORD, 'string'>
<T_IDENTIFIER, 'str'>
<T_OPERATOR, '='>
<T_STRING_LIT, '"Hello, World!"'>
<T_DELIMITER, ';'>
